lib/ship_compliant/add_update_brand.rb

```diff
@@ -37,8 +37,8 @@ class AddUpdateBrand
     #     ShipCompliant::AddUpdateBrand.product({
     #       # brand attributes
     #     }, update_mode: 'UpdateExisting')
-    def self.brand(brand, options = {})
-      result = ShipCompliant.client.call(:add_update_brand, {
+    def self.brand(brand, options: {}, configuration: :default)
```

Comment:
Oh I see. It's for consistency (hadn't read the rest of the PR when I made above comment)